src/mobile/src/components/Footer.jsx

```diff
@@ -0,0 +1,137 @@
+import { View, Text, Image, StyleSheet, Pressable, Linking } from 'react-native';
+import { AntDesign } from '@expo/vector-icons';
+import { Ionicons } from '@expo/vector-icons';
+
+import GitHubContributorCard from './GitHubContributorCard';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'column',
+    alignItems: 'center',
+    backgroundColor: '#121D59',
+    paddingHorizontal: 20,
+    paddingVertical: 10,
+    width: '100%',
+  },
+  stretch: {
+    width: 150,
+    height: 150,
+  },
+  heading: {
+    color: 'white',
+    fontSize: 25,
+    textTransform: 'uppercase',
+  },
+  subheading: {
+    color: 'white',
+    fontSize: 18,
+    paddingTop: 10,
+    paddingBottom: 6,
+  },
+  leftDivider: {
+    backgroundColor: 'white',
+    height: 2,
+    width: '60%',
+  },
+  column: {
+    flexDirection: 'column',
+    width: '100%',
+  },
+  row: {
+    flexDirection: 'row',
+    width: '100%',
+    justifyContent: 'space-between',
+  },
+  left: {
+    width: '50%',
+  },
+  right: {
+    width: '50%',
+    alignItems: 'flex-end',
+  },
+  rightDivider: {
+    backgroundColor: 'white',
+    height: 2,
+    width: '40%',
+  },
+  communityLogo: {
+    paddingVertical: 15,
+    marginRight: 10,
+  },
+  copyright: {
+    color: 'white',
+    fontSize: 15,
+    paddingVertical: 10,
+  },
+});
+
+const Footer = () => {
+  return (
+    <View style={styles.container}>
+      <Image style={styles.stretch} source={require('../assets/adaptive-icon.png')} />
+      <View style={styles.column}>
+        <View style={styles.row}>
+          <View style={styles.left}>
+            <Text style={styles.heading}>Docs</Text>
+            <View style={styles.leftDivider}></View>
+            <Pressable
+              onPress={() =>
+                Linking.openURL(
+                  'https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md'
```

Comment:
The link to this page is not found. Did you mean this link? `https://github.com/Seneca-CDOT/telescope/blob/master/src/docs/docs/getting-started/environment-setup.md`